src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomUapHiVI.java

```diff
@@ -0,0 +1,134 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.HiviUap;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class TestAdsFandomUapHiVI extends AdsFandomTestTemplate {
+
+  private static final double DEFAULT_STATE_ASPECT_RATIO = 4.0;
+  private static final double RESOLVED_STATE_ASPECT_RATIO = 10.0;
+  private static final double MOBILE_VIDEO_ASPECT_RATIO = 272.0 / 153.0;
+  private static final By TLB_SELECTOR = By.id("gpt-top-leaderboard");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "gpt-top-leaderboard" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)